docs/cody/capabilities/auto-edit.mdx

```diff
@@ -14,49 +14,9 @@
 
 ## Enabling auto-edit
 
-Eligible Cody Pro users will gradually be given access to the auto-edit feature. Once you have it, you will receive an in-editor notification.
+Auto-edit is enabled by default for Cody Pro and Enterprise users. You can opt out and switch back to **autocomplete** by selecting it from the suggestion mode in the Cody VS Code extension settings.
 
-<video width="100%" loop playsInline controls style={{ width: '100%', height: 'auto' }}>
-    <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/auto-edit-in-action-2025.mp4" type="video/mp4" />
-</video>
-
-### Auto-edit access for Enterprise customers
-
-Auto-edit is available for Enterprise customers with [Sourcegraph Cody Gateway](/cody/core-concepts/cody-gateway#sourcegraph-cody-gateway) access. Enabling the feature requires two steps:
-
-1. Site administrators must:
```

Comment:
@alexromano confirming: this section becomes invalid as it's enabled as default.

Reply:
* they'll still need to add fireworks as an allowed provider
* admins will be able to opt out of autoedits using the same flag
* developers won't get a notification to turn it on
* individual devs can also opt in/out of autoedits using the same settings page shown in that image. we should update the image when we have the new copy (it'll say "autoedits (beta)") and it is set as default